client/rest-high-level/src/test/java/org/elasticsearch/client/IndicesClientIT.java

```diff
@@ -381,6 +386,35 @@ public void testCloseNonExistentIndex() throws IOException {
         assertEquals(RestStatus.NOT_FOUND, exception.status());
     }
 
+    public void testRefresh() throws IOException {
+        {
+            final int numberOfShards = randomIntBetween(1, 5);
+            final int numberOfReplicas = randomIntBetween(1, 3);
```

Comment:
I think I changed my mind on this. Given that it's an integration test, let's remove the randomization which complicate things? Let's just set 1 shard and 0 replica?

Reply:
Yes, I also find the assert part is too complicated .. :)